Sources/MapboxMaps/Location/Pucks/Puck3D.swift

```diff
@@ -95,7 +95,7 @@ internal class Puck3D: Puck {
     }
 
     // MARK: Protocol Implementation
-    internal func updateLocation(location: Location) {
+    internal func updateLocation(location: Location, bearingType: BearingType) {
```

Comment:
shouldn't you be honoring bearing in this method?

Reply:
There was no existing use case of it which is why I decided not to make changes.

Reply:
It's used below as part of headingDirection